### DISCUSSION TITLE
Apple Health Support

### DISCUSSION THREAD

**Discussion by samuelleclerc** [klassm/walkingpad_macos_client#6] [Ideas]
Hi Matthias ✋

Thank you so much for sharing your code, I've really enjoyed using it in the last few weeks. 

I was wondering if you had any intention of adding support for syncing steps/walking workouts to Apple Health. I realize you probably have little time to devote to adding such a feature, but I wanted to ask just in case. If not, I'll look into learning Swift and see if I can't come up with something on my own :)

Thanks again!

**Comment by tobeehh** [comment]
great idea, overall a really nice application...
